package/src/components/Attachment/AudioAttachment.tsx

```diff
@@ -48,61 +51,63 @@ export const AudioAttachment = (props: AudioAttachmentProps) => {
     showSpeedSettings = false,
     testID,
   } = props;
+  const { changeAudioSpeed, pauseAudio, playAudio, seekAudio } = useAudioPlayer({ soundRef });
+  const isExpoCLI = !Sound.Player && Sound.initializeSound;
```

Comment:
I wouldn't check our SDK's flavor like this. In `optionalDependencies`, we also provide the SDK flavor which we can pass down here to determine which one is currently being used. 

Since this is relying on the API itself, if the API changes for one reason or another later on it'll break this hook.

Reply:
Good point. Thanks for pointing it out.

Reply:
I have fixed it.